src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -73,8 +73,8 @@ def _sanitize_oses(oses):
   The OS custom field no longer has the 'Chrome' value.
   It was replaced by 'ChromeOS'.
   """
-  for i in range(len(oses)):
-    if oses[i] == 'Chrome':
+  for i, os in enumerate(oses):
```

Comment:
nit: call this os_field so it's not conlficting with Python's os module.

Reply:
Done.